assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	height: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Space = styled.div`
+	flex: 0 1 ${ ( { isNext } ) => isNext ? 60 : 115 }px;
+`;
+
+const NavButton = styled.button`
+	display: block;
+	background: transparent;
+	border: none;
+	width: 40px;
+	height: 40px;
+	&:focus {
+		outline: 1px dotted -webkit-focus-ring-color
+	}
+`;
+
+const IconStyle = css`
+	border: solid ${ ( { theme } ) => theme.colors.fg.v1 };
+	border-width: 0 1px 1px 0;
+	display: inline-block;
+	padding: 3px;
+	cursor: pointer;
+	opacity: .25;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: inherit;
+	height: 25px;
+	width: 25px;
+	&:hover {
+		color: inherit;
```

Comment:
The hover effect should really be on the button and not the icon itself. Otherwise I can navigate through pages without the arrows turning white.